recipes/jfalcou-eve/all/conanfile.py

```diff
@@ -29,9 +29,9 @@ def _min_cppstd(self):
     @property
     def _compilers_minimum_version(self):
         return {
-            "gcc": "10.2",
+            "gcc": "11",
             "Visual Studio": "16.9",
-            "clang": "12",
+            "clang": "13",
             "apple-clang": "13",
```

Comment:
These new compiler requirements were working before, so now they should apply only to this new `v2022.03.0` version. Is there a reason to also increase these requirements for `v2021.10.0`? If not, you can return different values depending on the version

Reply:
for example:

```python
    @property
    def _compilers_minimum_version(self):
        if tools.Version(self.version) >= "2022.03.0":  # <- not tested, given the trailing `v` I'm not sure this works out-of-the-box
                return {
                    "gcc": "11",
                    "Visual Studio": "16.9",
                    "clang": "13",
                    "apple-clang": "13",
                    }
        else:
                return {
                    "gcc": "10.2",
                    "Visual Studio": "16.9",
                    "clang": "12",
                    "apple-clang": "13",
                    }
        
```